stripper/ze_rabanastre_royal_t5.cfg

```diff
@@ -793,3 +795,34 @@ modify:
 		"min_use_angle" "0.2"
 	}
 }
+
+;Patch some tp avoidence spots, this is a very rare spot to get to but it will still be needed due to avoiding nuke at the end.
+add:
+{
+	"classname" "trigger_teleport"
+	"model" "*15"
+	"origin" "-9832 4856 -11180"
+	"spawnflags" "4097"
+	"StartDisabled" "1"
+	"targetname" "TP_Bahamut_Boss"
+	"target" "TD_Bahamut_1"
+	"UseLandmarkAngles" "1"
+}
+
+add:
+{
+	"classname" "trigger_teleport"
+	"model" "*15"
+	"origin" "-9654 4856 -11180"
+	"spawnflags" "4097"
+	"StartDisabled" "1"
+	"targetname" "TP_Bahamut_Boss"
+	"target" "TD_Bahamut_1"
+	"UseLandmarkAngles" "1"
+}
+
+;Why is this here? this can screw up the players if they bhop to the laser boss.
+filter:
+{
+	"origin" "-6776 7672 -11412"
+}
```

Comment:
There are actually 3 of these physboxes in that hallway, and they appear to be intentional obstacles that fall when the func_breakables below them are broken. So I don't think these should be removed, unless there is a more serious issue here.